src/app/components/LiteSiteCta/index.tsx

```diff
@@ -0,0 +1,104 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '../../contexts/RequestContext';
+import styles from './index.styles';
+import { defaultTranslations } from './liteSiteConfig';
+
+type CtaLinkProps = {
+  isRtl: boolean;
+  href: string;
+  text: string;
+  fontVariant?: string;
+  showChevron?: boolean;
+  ignoreLiteExtension?: boolean;
+  className?: string;
+};
+
+const CtaLink = ({
```

Comment:
Just to jump in here. We could probably re-use the CallToActionLink component for display purposes, however click tracking won't work on `.lite` as it uses client-side side-effects like `useEffect`, which aren't available to us on Lite.

Reply:
We didn't initially go for this because the styling is fairly different. The existing CTA component uses text-decoration:underline whereas this component uses bottom border.

This is significant because text-decoration:underline ignores SVGs by default but bottom border doesn't. So we have to change the HTML structure so that the SVG doesn't get wrapped in the text element.

I have done a quick mock up of what this refactor looks like [here](https://github.com/bbc/simorgh/pull/12097). I think it could be further improved - e.g. if we moved the flex/ centre align styles out of this component and applied these via a className override. 

If you think it has legs then I can hand over for someone to develop whilst I'm on training next week?

(n.b. It could be a good candidate to review with UX and see if we can align the A11y behaviours/ styling across the site).